modules/model/shared/src/main/scala/observe/model/StepState.scala

```diff
@@ -16,35 +16,35 @@ enum StepState derives Eq:
   case Running             extends StepState
   case Paused              extends StepState
 
-  def canSetBreakpoint: Boolean = this match
+  lazy val canSetBreakpoint: Boolean = this match
```

Comment:
I was having a strange lazy initialization error regarding the derived `Eq`. Thought changing to `lazy val`s would fix it, but it didn't (`sbt clean` did). I decided to leave these as `lazy val`s anwyay.

Reply:
I wonder if these should be `inline`